README.md

```diff
@@ -70,12 +70,19 @@ Optional build tools:
 * [`graphviz`](https://graphviz.org/) - used by asciidoctor in the jetty-documentation build to produce various graphs
 * [`Docker`](https://www.docker.com/) - used to run some integration tests for testing third party integrations
 
-Eclipse Jetty will be built in `jetty-home/target/jetty-home`.
-
-The first build will take longer than subsequent builds as Maven will downloads all third party dependencies.
-
-The build tests do a lot of stress testing, and on some machines it is necessary to set the 
-file descriptor limit to greater than 2048 for the tests to all pass successfully.
+Once the build is complete, you can find the built Jetty Maven artifacts in your Maven local repository.
+Along with the following locations of note:
+
+Branches | Location                                                          | Description
+---------|-------------------------------------------------------------------|---------
+all      | `jetty-home/target/jetty-home-<ver>.tar.gz`                       | The Jetty Home standalone tarball
+`jetty-10.0.x` and `jetty-11.0.x` | `jetty-runner/target/jetty-runner-<ver>.jar`   | The Jetty Runner uber jar
```

Comment:
I still think we should not add `jetty-runner` here, but if we are going to keep it, split the branches (more readable even if duplicated) and add in the description the ee version they refer to (it's different).